src/tools/ResultsComparer/Program.cs

```diff
@@ -185,6 +196,75 @@ private static void ExportToCsv((string id, Benchmark baseResult, Benchmark diff
             Console.WriteLine($"CSV results exported to {csvPath.FullName}");
         }
 
+        private static void ExportToXml(List<string> slower, List<string> faster, FileInfo xmlPath)
```

Comment:
I think that it would be better to implement a method similar to `ExportToCsv` that uses strong types instead of strings. Strings need to be split and work until we change the way the results are printed.

Reply:
I wanted to be able to separate out the slower benchmarks from the faster benchmarks without having to implement the math twice. The XML file is only meant to be read by the xunit parser in Helix. Maybe I should change the parameter name and function name to make that clearer? This is also a stopgap until Drew implements this in the new visualization tool.

Reply:
> I wanted to be able to separate out the slower benchmarks from the faster benchmarks without having to implement the math twice

this information is available in the `conclusion` field. 


```cs
ExportToXml(notSame, args.XmlPath);

private static void ExportToXml((string id, Benchmark baseResult, Benchmark diffResult, EquivalenceTestConclusion conclusion)[] notSame, FileInfo xmlPath)
{
    using (XmlWriter writer = XmlWriter.Create(xmlPath.Open(FileMode.OpenOrCreate, FileAccess.Write, FileShare.Write)))
    {
        foreach (var slower in notSame.Where(x => x.conclusion == EquivalenceTestConclusion.Slower))
        {
            writer.WriteStartElement("test");
            writer.WriteAttributeString("name", slower.id);
            writer.WriteAttributeString("type", slower.baseResult.Type);
            writer.WriteAttributeString("method", slower.baseResult.Method);
            writer.WriteAttributeString("time", "0");
            writer.WriteAttributeString("result", "Fail");
            writer.WriteStartElement("failure");
            writer.WriteAttributeString("exception-type", "Regression");
            writer.WriteElementString("message", $"{slower.id} has regressed, was {slower.baseResult.Statistics.Median} is {slower.diffResult.Statistics.Median}");
            writer.WriteEndElement();
        }
    }
}
```

Reply:
Ok, sure.

Reply:
That worked really well. Thanks for the insight Adam. I'll be putting this part of the change as a separate PR.